contracts/USM.sol

```diff
@@ -46,12 +45,16 @@ contract USM is ERC20 {
      * @param ethIn Amount of wrapped Ether to use for minting USM.
      * @return USM minted
      */
-    function mint(uint256 ethIn) external returns (uint) {
-        require(eth.transferFrom(msg.sender, address(this), ethIn), "Eth transfer fail");
+    function mint(address from, address to, uint ethIn)
+        external override
+        onlyHolderOrDelegate(from, "Only holder or delegate")
+        returns (uint)
+    {
+        require(eth.transferFrom(from, address(this), ethIn), "Eth transfer fail");
         require(ethIn > MIN_ETH_AMOUNT, "0.001 ETH minimum");
```

Comment:
As with some of these checks before - I'm not strongly against it per se - but what's the benefit?  Unless there's a clear one I'd be inclined to scrap random checks like this.  (Especially since who knows, in the future 0.001 ETH might be like a million dollars)

Reply:
I'm always in favor of removing code :+1: